test-refined/shared/src/main/scala/zio/test/refined/CharInstances.scala

```diff
@@ -0,0 +1,47 @@
+package zio.test.refined
+
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.char.{Digit, Letter, LowerCase, UpperCase, Whitespace}
+import zio.random.Random
+import zio.test.Gen
+import zio.test.magnolia.DeriveGen
+
+object char extends CharInstances
+
+trait CharInstances {
+  private val alphaCharGen: Gen[Random, Char] =
+    Gen.weighted(Gen.char(65, 90) -> 26, Gen.char(97, 122) -> 26)
+
+  private val numericCharGen: Gen[Random, Char] =
+    Gen.weighted(Gen.char(48, 57) -> 10)
+
+  private val whitespaceChars: Seq[Char] =
+    (Char.MinValue to Char.MaxValue).filter(_.isWhitespace)
```

Comment:
Oh. May be move these to `Gen` companion object.

Reply:
Good idea. 👍